test/e2e/data/infrastructure-docker/v1beta1/bases/cluster-with-topology.yaml

```diff
@@ -24,8 +24,9 @@ spec:
         name: "md-0"
         replicas: ${WORKER_MACHINE_COUNT}
     variables:
```

Comment:
The idea is to have one variable which is defaulted and others which are set explicitly